<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchAndReplace</class>
 <widget class="QWidget" name="SearchAndReplace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Найти</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="editSearch"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="btnSearch">
     <property name="text">
      <string>Искать</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="btnNext">
     <property name="text">
      <string>Далее</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Заменить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="editReplace"/>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="btnReplace">
     <property name="text">
      <string>Заменить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="btnPrev">
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="checkRegister">
     <property name="text">
      <string>С учетом региста</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QCheckBox" name="checkRegEx">
     <property name="text">
      <string>Регулярные выражение</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
